domains/tracing/tracing.go

```diff
@@ -36,12 +36,13 @@ func (tm *tracingMiddleware) CreateDomain(ctx context.Context, session authn.Ses
 	return tm.svc.CreateDomain(ctx, session, d)
 }
 
-func (tm *tracingMiddleware) RetrieveDomain(ctx context.Context, session authn.Session, id string) (domains.Domain, error) {
+func (tm *tracingMiddleware) RetrieveDomain(ctx context.Context, session authn.Session, id string, withRoles bool) (domains.Domain, error) {
 	ctx, span := tracing.StartSpan(ctx, tm.tracer, "view_domain", trace.WithAttributes(
 		attribute.String("id", id),
+		attribute.Bool("get_roles", withRoles),
```

Comment:
Please change to with_roles
```suggestion
        attribute.Bool("with_roles", withRoles),
```